docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -7,16 +7,15 @@ weight: 10
 
 # Set up Go profiling in pull mode
 
-In pull mode, the Grafana Agent periodically retrieves profiles from Golang applications, specifically targeting the
+In pull mode, the collector, whether Grafana Alloy or Grafana Agent (legacy), periodically retrieves profiles from Golang applications, specifically targeting the
```

Comment:
Do you want to add a `(preferred)` here too for consistency? Or... do we need to call out `(preferred)` and `(legacy)` after the first definition (eg you could define it on line 12 and then just use Alloy/Agent everywhere else)?

Reply:
I think earlier and maybe once per procedure section? People might skip to the relevant sections.

Reply:
Ah good point

Reply:
```suggestion
In pull mode, the collector, whether Grafana Alloy (preferred) or Grafana Agent (legacy), periodically retrieves profiles from Golang applications, specifically targeting the
```